index.html

```diff
@@ -70,7 +70,7 @@ <h1>
             
             What are we forgetting? <br> <br> 
             
-            Oh! Onions. Yams. Avocados. Lettuce. Arugula (to some, “rocket”). Persian cucumbers, in addition to aforementioned “normal” cucumbers. Artichokes. Zucchinis. Pumpkins. Squash (what some cultures call pumpkins). Sweet potatoes and potato-potatoes. Jackfruit. Monk fruit. Fruit of the Loom. Fruits of our labor (this website). Sorrel. Pineapple. Mango. Gooseberries. Blackberries. Tomatoes. Heirloom tomatoes. Beets. Chives. Corn. Endive. Escarole, which, we swear, we’re vendors of organic produce, but if you asked us to describe what escaroles are...
```

Comment:
Remove some more lines

Reply:
ok

Reply:
done

Reply:
Oh my bad. can we add these lines again ?

Reply:
ok

Reply:
done

Reply:
Thanks